src/plugin.js

```diff
@@ -780,6 +812,10 @@ const contribAdsPlugin = function(options) {
     if (player.ads.tryToResumeTimeout_) {
       player.clearTimeout(player.ads.tryToResumeTimeout_);
     }
+
+    if (player.ads._trackChangeDuringAdsHandler) {
```

Comment:
This conditional isn't necessary, it's no problem to call removeEventListener even if you don't know if it's there.